File: Regular Expressions/Code/chapter1/1-26/recipe1-26.py
#!/usr/bin/python
import re
import sys

nargs = len(sys.argv)

if nargs > 1:

	mystr = sys.argv[1]

	r = re.compile( r'\n', re.M )

	newstr = r.sub( ', ', open( mystr ).read( ) )

	print newstr,

else:
	print 'Filename?  Anyone?  Anyone?',

File: Regular Expressions/Code/chapter1/1-26/recipe1-26.pl
#!/usr/bin/perl -w
use strict;

open( FILE, $ARGV[0] ) || die "Cannot open file!";

while ( <FILE> )
{
	my $line = $_;
	$line =~ s/\n/, /mg;
	print $line;
}

print "\n";

close( FILE );
